doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -315,36 +329,36 @@ set with:
 
 .. _bluestore-rocksdb-sharding:
 
-RocksDB Sharding
+rocksdb sharding
 ================
 
-Internally BlueStore uses multiple types of key-value data,
-stored in RocksDB.  Each data type in BlueStore is assigned a
-unique prefix. Until Pacific all key-value data was stored in
-single RocksDB column family: 'default'.  Since Pacific,
-BlueStore can divide this data into multiple RocksDB column
-families. When keys have similar access frequency, modification
-frequency and lifetime, BlueStore benefits from better caching
-and more precise compaction. This improves performance, and also
+internally bluestore uses multiple types of key-value data,
+stored in rocksdb.  each data type in bluestore is assigned a
+unique prefix. until pacific all key-value data was stored in
+single rocksdb column family: 'default'.  since pacific,
+bluestore can divide this data into multiple rocksdb column
+families. when keys have similar access frequency, modification
+frequency and lifetime, bluestore benefits from better caching
+and more precise compaction. this improves performance, and also
 requires less disk space during compaction, since each column
 family is smaller and can compact independent of others.
 
-OSDs deployed in Pacific or later use RocksDB sharding by default.
-If Ceph is upgraded to Pacific from a previous version, sharding is off.
+osds deployed in pacific or later use rocksdb sharding by default.
```

Comment:
case?

Reply:
All of this weird e e cummings/bell hooks-looking stuff is repaired in https://github.com/ceph/ceph/pull/51787.